uprotocol/transport/builder/uattributesbuilder.py

```diff
@@ -184,6 +211,8 @@ def build(self):
         attributes = UAttributes(id=self.id, type=self.type, priority=self.priority)
         if self.sink is not None:
             attributes.sink.CopyFrom(self.sink)
+        if self.source is not None:
```

Comment:
It will never be null as its a mandatory field

Reply:
Got it, fixed